.github/actions/deploy-frontend/action.yml

```diff
@@ -32,8 +35,13 @@ runs:
     - name: Download Frontend Artifact (React)
       uses: actions/download-artifact@v2
       with:
-        name: static_website_react-${{ github.run_id }}
-        path: frontend-react/build
+        name: static-website-react-${{ inputs.version }}
+        path: frontend-react
```

Comment:
Just to make sure I understand: the difference in the path is because we're now tar/gz-ing the build files ourselves (in `build/`) and then passing that resulting file to the archive-artifact action. Previously we passed a directory to the archive-artifact action.

Reply:
...and then we unpack it in the next step, doing so in the frontend-react directory so all the files go into `build/`

Reply:
Correct